src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student10/GymMemberController10.java

```diff
@@ -26,4 +28,10 @@ public Workout10 checkIn(@PathVariable int memberId, @RequestParam String time)
     public Workout10 checkOut(@PathVariable int memberId, @RequestParam String time) {
         return memberService.checkOut(memberId, time);
     }
+
+    @GetMapping("student10/api/v1/member/{memberId}/workout-history")
+    public List<Workout10> getWorkoutHistory(@PathVariable int memberId) {
```

Comment:
Does this return a dataset that matches the requirements for the assignment?  I'm looking for a structure that resembles something like this:

```
[
    {
        "memberId": 32,
        "timeUtc": "2022-01-02T19:48:28.000+00:00",
        "checkType": "CHECK_IN"
    },
    {
        "memberId": 32,
        "timeUtc": "2022-01-02T19:48:28.000+00:00",
        "checkType": "CHECK_OUT"
    }
   ...
]
```